modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc

```diff
@@ -289,14 +293,15 @@ Only persistent volumes (PVs) created from the default storage class are encrypt
 PVs created by using any other storage class are still encrypted, but the PVs are not encrypted with this key unless the storage class is specifically configured to use this key.
 ====
 
-<9> Optional: Only enable this option if your use case requires etcd key value encryption in addition to the control plane storage encryption that encrypts the etcd volumes by default. With this option, the etcd key values are encrypted but not the keys.
+<10> Optional: You can select additional custom security groups to use in your cluster, if you have already created the security groups and associated them with the VPC you selected for this cluster. See the requirements for link:https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/prepare_your_environment/rosa-sts-aws-prereqs#rosa-security-groups_rosa-sts-aws-prereqs[Security groups] for more details.
```

Comment:
Same comment as another file where I suggested to break the first sentence into two sentences.

Also same comment about the hard link

Reply:
Split the sentences, coming back to the hard link.

Reply:
Fixed hard link to soft reference and additional resources xref.